ci-operator/step-registry/servicemesh/sail-operator/e2e-lpinterop/servicemesh-sail-operator-e2e-lpinterop-commands.sh

```
@@ -26,36 +26,27 @@
-    if [[ $MAP_TESTS == "true" ]]; then
-        results_file="${1}"
-        echo "Patching Tests Result File: ${results_file}"
-        if [ -f "${results_file}" ]; then
-            install_yq_if_not_exists
-            echo "Mapping Test Suite Name To: ServiceMesh-lp-interop"
-            yq eval -px -ox -iI0 '.testsuites.testsuite[]."+@name" = "ServiceMesh-lp-interop"' "${results_file}" || echo "Warning: yq failed for ${results_file}, debug manually" >&2
-        fi
-    fi
-}
+# login for interop
+if test -f ${SHARED_DIR}/kubeadmin-password
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)